libsweep/examples/example.cc

```diff
@@ -14,7 +14,7 @@ int main() try {
     auto scan = device.get_scan();
 
     for (auto sample : scan.samples) {
-      std::cout << "angle " << sample.angle << " distance " << sample.distance << " strength " << sample.signal_strength << "\n";
+      std::cout << "Angle: " << sample.angle/1000.f << ", Distance: " << sample.distance << ", Signal Strength: " << sample.signal_strength << "\n";
```

Comment:
Can you clang-format all your changes - there is a `.clang-format` style file in the repo.

Work with `clang-format-3.8` for sure - other versions might introduce some differences - we have to see.

If this is too much of a hassle for you I can do it before merging.